spec/index.bs

```diff
@@ -5888,6 +5889,19 @@ dictionary GPUImageCopyTextureTagged : GPUImageCopyTexture {
         2d canvases are [[html#premultiplied-alpha-and-the-2d-rendering-context|always premultiplied]],
         while WebGL canvases can be controlled via <l spec=html>[=WebGLContextAttributes=]</l>.
         {{ImageBitmap}} premultiplication can be controlled via {{ImageBitmapOptions}}.
+
+    : <dfn>originBottomLeft</dfn>
+    ::
+        Describes whether the origin (0, 0) to copy from source image is bottom left corner, or not.
+       
+        If this option is set to `true` and the origin (0, 0) of {{GPUImageCopyExternalImage/source}} is top left corner,
+        the source values must be flipped along the vertical axis to change the origin (0, 0) to bottom left corner.
+
+        Note:
+        If {{GPUImageCopyTextureTagged/originBottomLeft}} matches the source image, no conversion occurs.
+        Origin (0, 0) of 2d canvases are [[html#2dcontext|always top left corner]]. Origin (0, 0) of webgl
+        canvases are always top left corner. Origin (0, 0) of webgpu canvases are [[#coordinate-systems|always top left corner]].
```

Comment:
WebGL has a hint https://www.khronos.org/registry/webgl/specs/latest/1.0/#2.4 here about the origin (0, 0). But I'm not sure whether we could have a link for it.

Reply:
I don't think it's necessary to mention different canvas types or different source types. I think we can tread canvases, ImageBitmaps, etc. all the same in that they just contain some (intrinsically oriented) 2d image data. We want to take advantage of that abstraction to simply describe whether we read bottom-up or top-down.